src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -64,6 +64,12 @@ public enum PropertyType {
 		 */
 		BE,
 
+		/**
+		 * Indicates that the condition is in a form of <code>something is/are something</code>,
+		 * also possibly in the negated form
+		 */
+		IS,
+
```

Comment:
this is just BE, why does it need to be added

Reply:
BE doesn't equal is/are

Reply:
well is/are are conjugated forms of "to be", so BE makes sense

Reply:
Is this blocking to not have both enums?

Reply:
Yeah, either rename BE or keep BE (i prefer keeping)
It's just confusing to have two types that do and mean the exact same thing

Reply:
renaming would be a breaking change!

Reply:
Not renaming, it's adding the proper terminology.

Reply:
As already mentioned, BE is the proper terminology. No need to add another that does the same thing.